pyk/src/pyk/utils.py

```diff
@@ -688,6 +688,10 @@ def add_file_contents(self, input: str, arcname: Path) -> None:
             ntf.flush()
             self.add_file(Path(ntf.name), arcname)
 
+    def add_request(self, req_name: str) -> None:
+        self.add_file_contents(req_name, Path(f'sequence/{self._command_id}'))
```

Comment:
Let's make it easier to sort these files:
```suggestion
        self.add_file_contents(req_name, Path(f'sequence/{self._command_id:03}'))
```